bun.lock

```diff
@@ -1,5 +1,6 @@
 {
   "lockfileVersion": 1,
+  "configVersion": 0,
```

Comment:
????????????????????????????????????????????? how old is your bun version that it is not `{ "configVersion": 1 }`